app/main.py

```diff
@@ -1 +1,27 @@
-# write your code here
+import os
+import shutil
+
+
+def move_file(command: str) -> None:
+    parts = command.split()
+    if len(parts) != 3 or parts[0] != "mv":
+        raise ValueError(
+            "Invalid command format. Expected format: 'mv source destination'"
+        )
+
+    source_path = parts[1]
+    destination_path = parts[2]
+
+    if not os.path.exists(source_path):
+        raise FileNotFoundError(f"Source file {source_path} does not exist")
+
+    if destination_path.endswith("/"):
+        os.makedirs(destination_path, exist_ok=True)
```

Comment:
Issue: The code attempts to create a directory without checking if it already exists, which could lead to an error. According to the checklist, you should handle the case where the directory already exists.